lib/pages/new_group/new_group_chat_info.dart

```diff
@@ -212,37 +178,75 @@ class NewGroupChatInfo extends StatelessWidget {
+          child: ValueListenableBuilder(
+            valueListenable: newGroupController.avatarNotifier,
+            builder: (context, value, child) {
+              if (value == null) {
+                return child!;
+              }
+              return ClipOval(
+                child: SizedBox.fromSize(
+                  size: const Size.fromRadius(28),
+                  child: AssetEntityImage(
+                    value,
+                    thumbnailSize: const ThumbnailSize(56, 56),
+                    fit: BoxFit.cover,
+                    loadingBuilder:(context, child, loadingProgress) {
+                      if (loadingProgress != null && loadingProgress.cumulativeBytesLoaded != loadingProgress.expectedTotalBytes) {
+                        return const Center(
+                          child: CircularProgressIndicator.adaptive(),
+                        ); 
+                      }
+                      return child;
+                    },
+                    errorBuilder: (context, error, stackTrace) {
+                      return const Center(
+                        child: Icon(Icons.error_outline),
+                      );
+                    },
+                  ),
+                ));
+            },
+            child: Icon(
+              Icons.camera_alt_outlined,
+              color: Theme.of(context).colorScheme.surface,
+            ),
           ),
-        ),
-      ),
-    );
+        ));
   }
 
   Widget _buildGroupNameTextFieid(BuildContext context) {
     return Padding(
       padding: const EdgeInsets.symmetric(horizontal: 8.0),
-      child: TextField(
-        controller: newGroupController.groupNameTextEditingController,
-        focusNode: newGroupController.groupNameFocusNode,
-        decoration: InputDecoration(
-          border: OutlineInputBorder(
-            borderSide: BorderSide(color: Theme.of(context).colorScheme.shadow),
-          ),
-          labelText: L10n.of(context)!.widgetName,
-          labelStyle: Theme.of(context).textTheme.bodySmall?.copyWith(
-            fontSize: 16,
-            letterSpacing: 0.4,
-            color: Theme.of(context).colorScheme.onSurface,
-          ),
-          hintText: L10n.of(context)!.enterGroupName,
-          hintStyle: Theme.of(context).textTheme.bodyLarge?.copyWith(
-            letterSpacing: -0.15,
-            color: Theme.of(context).colorScheme.onSurface,
-          ),
-          contentPadding: const EdgeInsets.all(16.0),
-        ),
+      child: ValueListenableBuilder(
+        valueListenable: newGroupController.createRoomStateNotifier,
+        builder: (context, value, child) {
+          return TextField(
+            controller: newGroupController.groupNameTextEditingController,
+            focusNode: newGroupController.groupNameFocusNode,
+            enabled: value?.fold(
+              (l) => true, 
+              (r) => !(r is CreateNewGroupChatLoading || r is UploadContentLoading)
```

Comment:
- IMO: If you are a reader you will not understand what `l` or `r` is. you should define more clearly

Reply:
done, there are `left` and `right`